service/key/key.go

```diff
@@ -69,7 +70,7 @@ func EtcdSecret(installation string, obj interface{}) string {
 }
 
 func IsCAPIManagementCluster(provider string) bool {
-	return provider == "cloud-director" || provider == "gcp" || provider == "openstack" || provider == "vsphere"
+	return provider == "capa" || provider == "cloud-director" || provider == "gcp" || provider == "openstack" || provider == "vsphere"
```

Comment:
Added Capa as a provider

Reply:
https://github.com/giantswarm/config/blob/d358a22c5e56840c8ca0581ca77c8d7a76973683/installations/tornado/config.yaml.patch#L6

Reply:
I guess it's more readable and convenient if we implement a grouping switch case!

Reply:
I went with a slice because i really do not think a switch is more readable